server/index.js

```diff
@@ -1,57 +1,80 @@
 //Basic requirements
-const express = require('express')
-const bodyParser = require('body-parser')
-const cors = require('cors')
+const express = require("express");
+const bodyParser = require("body-parser");
+const cors = require("cors");
 const app = express();
 
 //Mongo Atlas requirements
-const mongoclient = require('mongodb').MongoClient;
-const objectid = require('mongodb').ObjectID;
-const uri = "mongodb+srv://<username>:<password>@cluster0-pf0b2.mongodb.net/test?retryWrites=true&w=majority";
-const userModel = require('./user-schema');
+const mongoclient = require("mongodb").MongoClient;
+const objectid = require("mongodb").ObjectID;
+const uri =
+  "mongodb+srv://greg34910:cookie123@cluster0-pf0b2.mongodb.net/test?retryWrites=true&w=majority";
```

Comment:
You are adding your personal mongoDB account info. Can you make this generic?

Reply:
Oops! Nice catch, will change that.